### PR TITLE
[[ Docs ]] Rebuild extension docs where possible if they have changed

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

File: Toolset/libraries/revidelibrary.8.livecodescript [hunk 1]
```diff
@@ -7946,7 +7946,16 @@ private function escape pString, pConvertLineEndings
...
 
-private function revIDEGetDocsAPIData
+private function __getLCBSourceFile pFolder
+   set the defaultFolder to pFolder
+   # Isolate the source file
+   local tSource
+   put the files into tSource
+   filter tSource with "*.lcb"
+   return tSource
+end __getLCBSourceFile
```
Comment (peter-b) [inline Toolset/libraries/revidelibrary.8.livecodescript:7959]: Is it okay not to restore the `defaultfolder` here?

Comment (livecodeali) [inline Toolset/libraries/revidelibrary.8.livecodescript:7959]: No! Thanks.

**Comment by livecodeali** [comment]
I'm getting occasional start up errors in this branch, so WIP while I figure it out.

It is plausible that the unrestored default folder was causing the problem.

**Comment by livecodefraser** [comment]
Did restoring the default folder seem to fix the issue?

**Comment by livecodeali** [comment]
I'm not sure - it was intermittent at the time. Might be best to merge it and see so it will be being tested continually. Can always revert if it causes problems.

**Comment by peter-b** [comment]
@livecode-vulcan review ok 5a6348a

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 5a6348a1818dcc768320eff23053799438c0dc71

**Pull request merged**